streams/src/main/java/org/apache/kafka/streams/Streams.java

```diff
@@ -34,20 +34,20 @@
 import java.util.concurrent.atomic.AtomicInteger;
 
 /**
- * Kafka Streaming allows for performing continuous computation on input coming from one or more input topics and
+ * Kafka Streams allows for performing continuous computation on input coming from one or more input topics and
```

Comment:
Just Streams? If you want this to be Kafka Streams, you need to change the paragraphs below to be consistent.